src/parser/user-extractor-module.ts

```diff
@@ -42,19 +42,31 @@ export class UserExtractorModule implements Module {
     return sortedPriceLabels[0];
   }
 
+  _getAssigneeCount(issue: GitHubIssue) {
+    return 1 / (issue.assignees?.length || 1);
```

Comment:
Should we use a math library? Dealing with floats seems risky.

Reply:
Yes you are right, I always forget JS is full of surprises, will change it.